token/cli/src/main.rs

```diff
@@ -193,8 +193,6 @@ pub fn mint_decimals_arg<'a, 'b>() -> Arg<'a, 'b> {
         .takes_value(true)
         .value_name("MINT_DECIMALS")
         .validator(is_mint_decimals)
-        .requires(SIGN_ONLY_ARG.name)
```

Comment:
This is currently done through `offline::ArgsConfig` for the appropriate commands. This will relax the restriction for online mode though, so that you can still specify the decimals and all that when finally sending the transaction, which makes for a nicer UX if you ask me

Reply:
yeah was mainly hoping to disallow doing the final sign+submit invocation of an offline transaction without the identical cli args used for the sign-only invocation(s)

Reply:
That's what we're doing here by removing the sign only requirement from these other args, so that submission can keep using those args too